jekyll/_cci2/upgrade-2.14.1-to-2.15.0.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: classic-docs
+title: "Upgrading from CircleCI Enterprise Installation 2.14.01 to 2.15.0"
+category: [administration]
+order: 12
+description: "How to upgrade from CircleCI Enterprise Installation 2.14.01 to 2.15.0 when using externalized MongoDB"
+---
+
+This document provides instructions for System Administrators upgrading an existing CircleCI Enterprise Installation 2.14.01 to 2.15.0 in the following sections:
+
+* TOC
+{:toc}
+
+**Notes:**
```

Comment:
So, this is a prerequisite? You upgrade CircleCI first, then you can upgrade Mongo? If so, change this to Prerequisite and update the title of this document to Upgrading MongoDB from 3.2 to 3.6.